src/qt/transactionrecord.cpp

```diff
@@ -19,14 +19,10 @@
 
 /* Return positive answer if transaction should be shown in list.
  */
-bool TransactionRecord::showTransaction(const CWalletTx& wtx)
+bool TransactionRecord::showTransaction()
```

Comment:
I'd rather let the argument be or completely remove the function.

Reply:
Agree on completely removing `TransactionRecord::showTransaction()` which should also make the private member `showTransaction` in `TransactionNotification` unnecessary.

Reply:
done